src/components/Divider/Divider.tsx

```diff
@@ -43,6 +43,9 @@ class Divider extends UIComponent<Extendable<IDividerProps>, any> {
     /** Shorthand for primary content. */
     content: customPropTypes.contentShorthand,
 
+    /** A divider can be fitted, without any space above or below it.  */
+    divider: PropTypes.bool,
```

Comment:
this should be name as `fitted`. Also, please, make sure that `IDividerProps` reflects the changes you've introduced for the API

Reply:
Ah, I missed it. Fixing it right away.